File: bl-central-server/bl-web-app/src/main/java/org/jesperancinha/logistics/web/services/SquareBoundary.java
package org.jesperancinha.logistics.web.services;

import lombok.Builder;

import java.math.BigDecimal;

public record SquareBoundary(BigDecimal westLatitude,
    BigDecimal eastLatitude,
    BigDecimal northLongitude,
    BigDecimal southLongitude) {
    @Builder
    public SquareBoundary{
    }
}

File: bl-central-server/bl-web-app/src/main/kotlin/org/jesperancinha/logistics/web/services/SquareBoundary.kt
package org.jesperancinha.logistics.web.services

import java.math.BigDecimal

data class SquareBoundary(
    val westLatitude: BigDecimal,
    val eastLatitude: BigDecimal,
    val northLongitude: BigDecimal,
    val southLongitude: BigDecimal
)